aries-site/src/pages/components/rangeinput.mdx

```diff
@@ -1,17 +1,56 @@
 import { Example } from '../../layouts';
-import { RangeInputExample } from '../../examples';
+import {
+  RangeInputExample,
+  RangeInputInFormExample,
+  RangeInputDescriptionExample,
+} from '../../examples';
 
 <Example
   code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/rangeinput/RangeInputExample.js"
   docs="https://v2.grommet.io/rangeinput?theme=hpe#props"
-  details={[
-    `The RangeInput component is a slider control that 
-    provides a handle the user can move to make changes 
-    to values. It is important that the slider provides 
-    a value displayed to communicate with the user. This
-    help ensure conficence in the use of the control.`,
-  ]}
   figma="https://www.figma.com/file/BqCjvjc0rECQ4Ln2QhyjNi/HPE-Range-Input-Component?node-id=1%3A11"
 >
   <RangeInputExample />
 </Example>
+
+## Guidance
+
+A RangeInput allows a user to adjust the value by either decreasing or increasing the value.
+The user can adjust the value by moving the thumb across the track.
+
+- The RangeInput should be clear within the label for the user to know what value they are choosing.
+- There should be the min and the max on each side of the RangeInput so the user is aware of these values.
+- Do not use ranges that are to large i.e 1-10000 or ranges that are very small i.e 1-3
+
+### Usage
```

Comment:
Would like to hear about more tangible use cases / applications (e.g. volume, lighting dimmer, other more enterprise focussed use cases)

Reply:
Ill add some more to `Usage` for this!